test/test-rotatesTetrominos.js

```diff
@@ -13,6 +13,7 @@ describe("Class tetramino", function () {
             -----
         `);
         let lTetromino = new Tetromino(typeOfTetromino, phase, coordinatesTetromino)
+        let rotateTetromino = finalTetraminoRotate(lTetromino.typeOfTetromino, lTetromino.typeOfTetromino, lTetromino.initialCoordinates)
```

Comment:
finalTetraminoRotate - you don't have such method

Reply:
Shoud I improve the function rotateTetramino?